packages/grafana-ui/src/components/Forms/InlineField.story.tsx

```diff
@@ -11,40 +11,74 @@ import mdx from './InlineField.mdx';
 const meta: ComponentMeta<typeof InlineField> = {
   title: 'Forms/InlineField',
   component: InlineField,
+  argTypes: {
+    label: { control: { type: 'text' } },
+    labelWidth: { control: { type: 'number', min: 12 } },
+    tooltip: { control: { type: 'text' } },
+    error: { control: { type: 'text' } },
+  },
   parameters: {
     docs: {
       page: mdx,
     },
+    controls: {
+      exclude: ['htmlFor', 'className', 'children'],
+    },
   },
 };
 
-export const basic: ComponentStory<typeof InlineField> = () => {
+export const basic: ComponentStory<typeof InlineField> = (args) => {
   return (
-    <InlineField label="Inline field">
+    <InlineField {...args}>
       <Input placeholder="Inline input" />
     </InlineField>
   );
 };
 
-export const withTooltip: ComponentStory<typeof InlineField> = () => {
+basic.args = {
+  label: 'Inline field',
+  transparent: false,
+  grow: false,
+  shrink: false,
+  disabled: false,
+  interactive: false,
+  loading: false,
+  required: false,
```

Comment:
Interestingly enough `required` doesn't seem to work for `InlineField` (controls just made me realize that 🎉 ), I think it is missing some of the same logic as `Field`. This is unrelated to your changes, but did you want to fix that in this branch as well or I can create a separate issue for that? 😄

Reply:
Sounds good @gefgu ! I have created the issue here: https://github.com/grafana/grafana/issues/54848 please comment on it when you want to pick it up and I can assign it to you 😄